app/bnapi/bnapi.go

```diff
@@ -166,9 +167,19 @@ func MockBeaconNode(validators map[string]eth2v1.Validator) http.Handler {
 	}))
 
 	router.Handle("/eth/v1/config/deposit_contract", logHandler(func(writer http.ResponseWriter, request *http.Request) {
+		addressBytes, err := hex.DecodeString("07b39f4fde4a38bace212b546dac87c58dfe3fdc")
```

Comment:
why are we using static values?

Reply:
this is a mock response, only needed to make go-eth2-client initialize correctly